jobserver/views/job_requests.py

```diff
@@ -215,9 +221,24 @@ def get(self, request, *args, **kwargs):
             )
         )
 
+        # build up is_missing_updates to define if we've not seen the backend
```

Comment:
Is it worth moving the logic to calculate `is_missing_updates` to a method on the model?